src/components/VideoAsset.vue

```diff
@@ -15,28 +15,43 @@
     :should-wrap="!!deviceFrame"
     :device="deviceFrame"
   >
-    <video
-      ref="video"
-      :controls="showsControls"
-      :data-orientation="orientation"
-      :autoplay="autoplays"
-      :poster="normalisedPosterPath"
-      :muted="muted"
-      :width="optimalWidth"
-      playsinline
-      @loadedmetadata="setOrientation"
-      @playing="$emit('playing')"
-      @pause="$emit('pause')"
-      @ended="$emit('ended')"
-    >
-      <!--
-        Many browsers do not support the `media` attribute for `<source>` tags
-        within a video specifically, so this implementation for dark theme assets
-        is handled with JavaScript media query listeners unlike the `<source>`
-        based implementation being used for image assets.
-      -->
-      <source :src="normalizePath(videoAttributes.url)">
-    </video>
+    <div>
+      <span
+        v-if="alt"
+        :id="alt ? altTextId : null"
+        class="visuallyhidden"
+      >
+        {{ $t('formats.colon', {
+          content: $t('video.description')
+        }) + alt }}
```

Comment:
You likely have more expertise in this area than me, but I'm wondering if we should be concatenating these two strings together like this as opposed to making the whole string localizable in the following way in case other languages require these phrases to be combined in different ways:

```vue
{{ $t('video.description', { alt }) }}
```

where the key in English might look like:

```json
"video": {
  "description": "Content description of this video: {alt}"
```

I'm definitely not an expert on this, so I'll defer to your judgement here.